### DISCUSSION TITLE
Mantine and SSR rendering in Next.JS

### DISCUSSION THREAD

**Discussion by Xyzjesus** [mantinedev/mantine#4977] [Q&A]
hey! where can I get more info about ssr rendering with mantine?

**Comment by rtivital** [comment]
It works fine, follow Next.js getting started guide and you are good. What question did you have?

**Reply by Xyzjesus** [reply]
hi, thanks for answer! I've found docs and latest discussing about ssr and am I right that ssr works only with app routing?

**Reply by rtivital** [reply]
No, ssr works both with pages and app router.

**Reply by Xyzjesus** [reply]
I think it doesn't work in my project with pages router, only client side I think...
